homeassistant/components/remote/__init__.py

```diff
@@ -38,8 +41,12 @@
 
 SCAN_INTERVAL = timedelta(seconds=30)
 SERVICE_SEND_COMMAND = 'send_command'
+SERVICE_SEND_COMMANDS = 'send_commands'
 SERVICE_SYNC = 'sync'
 
+DEFAULT_NUM_REPEATS = '1'
```

Comment:
I'm curious, what do you mean by flakiness? If you're talking about the number of repeats parameter, that has nothing to do with flakiness. The intent of that it to allow you to repeat a command X number of times. For example, if you wanted to increase the volume by 5, you could send the volume up command with a number of repeats of 5.